requirements.txt

```diff
@@ -1,2 +1,3 @@
-PyYAML
-websocket-client
\ No newline at end of file
+PyYAML==6.0
+websockets==10.4
```

Comment:
Why is `websockets` needed? I think we just need `websocket-client`.

Reply:
you're right. I thought `websocket-client` installed `websockets` (I found it in my venv's `pip freeze`) and hence pinned both to be explicit.

but I verified again in a fresh venv, `websockets` is not installed by `websocket-client` 👍🏽